cli/chaosli/cmd/create.go

```diff
@@ -560,6 +574,49 @@ func getCount() *intstr.IntOrString {
 	return &wrappedResult
 }
 
+func getPulse() *v1beta1.DisruptionPulse {
+	validator := func(val interface{}) error {
+		if str, ok := val.(string); ok {
+			_, err := time.ParseDuration(str)
+			if err != nil {
+				return err
+			}
+
+			duration := v1beta1.DisruptionDuration(str)
+			if duration.Duration().Milliseconds() < 500 {
```

Comment:
We repeat `500` four times in this file. Could that be pulled out to a constant somewhere and referred to? That would make altering it in the future easier